lib/components/user/saved-trip-list.js

```diff
@@ -44,13 +67,24 @@ class SavedTripList extends Component {
           {accountLink}
           <h1>My saved trips</h1>
           <p>Click on a saved trip below to modify it.</p>
-          <ButtonGroup vertical block>
-            {trips.map((trip, index) => (
-              <Button key={index} onClick={this._handleTripSelect(trip)} style={{textAlign: 'left'}}>
+
+          {trips.map((trip, index) => (
+            <Panel key={index}>
```

Comment:
I prefer the trip Panel to be its own component (doesn't necessarily need its own file).

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/231/commits/19709bed5ca9d77f7915003870a9f4c233ff33ca